lib/reporters/html/index.js

```diff
@@ -63,6 +63,8 @@ PostmanHTMLReporter = function (newman, options) {
                     // set base values for overall response size and time values
                     _.set(executionMeans, executionId, { time: { sum: 0, count: 0 }, size: { sum: 0, count: 0 } });
 
+                    currentExecution.response = currentExecution.response.toJSON();
```

Comment:
This mutates the executions array in `summary.run.executions`, and therefore is not an acceptable solution.